var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -84,6 +90,10 @@ class Qt(Package):
     depends_on("libmng")
     depends_on("jpeg")
     depends_on("icu4c")
+    # FIXME:
+    # depends_on("freetype", when='@5.8:') and '-system-freetype'
```

Comment:
Depending on the version of freetype, there can be two include directories that are necessary, one for the general headers, and another for `ft2build.h`. I don't know what Spack's freetype generates or what Qt5 expects. Fedora seems to [move the libraries](https://src.fedoraproject.org/cgit/rpms/qt5-qtbase.git/tree/qt5-qtbase.spec#n380) to avoid accidental compilation. [Here](https://src.fedoraproject.org/cgit/rpms/qt5-qtbase.git/tree/qt5-qtbase.spec#n428) is the `configure` line used. Seems they just say `-fontconfig` and nothing about freetype.